src/libraries/Microsoft.Extensions.Configuration.Xml/src/XmlConfigurationElement.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.Configuration.Xml
+{
+    internal class XmlConfigurationElement
+    {
+        public string ElementName { get; }
+
+        public string Name { get; }
+
+        public string LineInfo { get; }
+
+        /// <summary>
+        /// The children of this element
+        /// </summary>
+        public List<XmlConfigurationElement>? Children { get; set; }
```

Comment:
It seems like we haven't enabled nullable for this project yet.

Reply:
OK I'll remove the annotations